code/modules/mob/mob_helpers.dm

```diff
@@ -409,7 +409,7 @@ GLOBAL_LIST_INIT(intents, list(INTENT_HELP,INTENT_DISARM,INTENT_GRAB,INTENT_HARM
 		return
 	else
 		if(alert(src, "You sure you want to sleep for a while?", "Sleep", "Yes", "No") == "Yes")
-			SetSleeping(40 SECONDS) //Short nap
+			SetSleeping(40 SECONDS, voluntary = TRUE) //Short nap
```

Comment:
this else isn't needed as the statement above it returns.

Reply:
Removed.